### DISCUSSION TITLE
qrz lookup

### DISCUSSION THREAD

**Discussion by g7kse** [magicbug/Cloudlog#964] [Q&A]
Is it me or has qrz.com started restricting the lookup to fill in blanks? It was working but noticed it wasn't filling anything in. I checked config.php and it was set to hamqth (duh) so updated that and neither of them are pulling the data now. 

I've set the logging to 1 in config.php but nothing showing up (yet). Wondering if it was me or if the service has now been made a fully paid for service (which of course I haven't paid up because I'm a tightwad)

Alex

**Comment by magicbug** [comment]
Not noticed any change here but I have a paid qrz subscription

**Reply by g7kse** [reply]
It's probably me. I should really pay up and stop being so tight
